v2/spanner-to-sourcedb/src/test/java/com/google/cloud/teleport/v2/templates/SpannerToSourceDbNewTablesIT.java

```diff
@@ -0,0 +1,160 @@
+  private SubscriptionName subscriptionName;
+
+  /**
+   * Setup resource managers and Launch dataflow job once during the execution of this test class.
+   *
+   * @throws IOException
+   */
+  @Before
+  public void setUp() throws IOException {
+    skipBaseCleanup = true;
+    synchronized (SpannerToSourceDbNewTablesIT.class) {
+      testInstances.add(this);
+      if (jobInfo == null) {
+        spannerResourceManager =
+            createSpannerDatabase(SpannerToSourceDbNewTablesIT.SPANNER_DDL_RESOURCE);
+        spannerMetadataResourceManager = createSpannerMetadataDatabase();
+
+        jdbcResourceManager = MySQLResourceManager.builder(testName).build();
+
+        createMySQLSchema(
+            jdbcResourceManager, SpannerToSourceDbNewTablesIT.MYSQL_SCHEMA_FILE_RESOURCE);
+
+        gcsResourceManager =
+            GcsResourceManager.builder(artifactBucketName, getClass().getSimpleName(), credentials)
+                .build();
+        createAndUploadShardConfigToGcs(gcsResourceManager, jdbcResourceManager);
+        gcsResourceManager.uploadArtifact(
+            "input/session.json", Resources.getResource(SESSION_FILE_RESOURCE).getPath());
+        pubsubResourceManager = setUpPubSubResourceManager();
+        subscriptionName =
+            createPubsubResources(
+                getClass().getSimpleName(),
+                pubsubResourceManager,
+                getGcsPath("dlq", gcsResourceManager).replace("gs://" + artifactBucketName, ""));
+        jobInfo =
+            launchDataflowJob(
+                gcsResourceManager,
+                spannerResourceManager,
+                spannerMetadataResourceManager,
+                subscriptionName.toString(),
+                null,
+                null,
+                null,
+                null);
+      }
+    }
+  }
+
+  /**
+   * Cleanup dataflow job and all the resources and resource managers.
+   *
+   * @throws IOException
+   */
+  @AfterClass
+  public static void cleanUp() throws IOException {
+    for (SpannerToSourceDbNewTablesIT instance : testInstances) {
+      instance.tearDownBase();
+    }
+    ResourceManagerUtils.cleanResources(
+        spannerResourceManager,
+        jdbcResourceManager,
+        spannerMetadataResourceManager,
+        gcsResourceManager,
+        pubsubResourceManager);
+  }
+
+  @Test
+  public void spannerToSourceNewTables() throws InterruptedException {
+    assertThatPipeline(jobInfo).isRunning();
+    // Write row in Spanner
+    writeRowInSpanner();
+    // Assert events on Mysql
+    assertRowInMySQL();
+  }
+
+  private void writeRowInSpanner() {
+    Mutation m1 =
+        Mutation.newInsertOrUpdateBuilder("Users").set("id").to(1).set("name").to("A").build();
+    spannerResourceManager.write(m1);
+
+    Mutation m2 =
+        Mutation.newInsertOrUpdateBuilder("Users2").set("id").to(2).set("name").to("B").build();
+    spannerResourceManager.write(m2);
+  }
+
+  private void assertRowInMySQL() throws InterruptedException {
```

Comment:
Skipped tables get written to 'skip' sub-directory in DLQ directory. Can we add a check for that there exists a file there?